shared/actions/chat/index.js

```diff
@@ -1006,6 +1080,14 @@ function* _exitSearch() {
 function* chatSaga(): SagaGenerator<any, any> {
   yield Saga.safeTakeEvery('app:changedFocus', _changedFocus)
   yield Saga.safeTakeEvery('chat:appendMessages', _sendNotifications)
+  yield Saga.safeTakeEvery('chat:clearMessages', _clearConversationMessages)
+  yield Saga.safeTakeEvery(['chat:appendMessages', 'chat:prependMessages'], _storeMessageToEntity)
+  yield Saga.safeTakeEvery(['chat:appendMessages', 'chat:prependMessages'], _findMessagesToDelete)
+  yield Saga.safeTakeEvery(['chat:appendMessages', 'chat:prependMessages'], _findMessageUpdates)
+  yield Saga.safeTakeEvery(['chat:updateMessage', 'chat:updateTempMessage'], _updateMessageEntity)
```

Comment:
that pattern would also clean up the hacky code we have for attachments where we have local state and state inside the messages.